meshtastic/interfaces/ble/receive_service.py

```
@@ -0,0 +1,728 @@
+                return True, False
+            return False, False
+        except (BleakDBusError, BLEClient.BLEError) as exc:
+            if iface._handle_read_loop_disconnect(repr(exc), client):
+                return True, False
+            return True, True
+        except (SystemExit, KeyboardInterrupt):  # pylint: disable=W0706
+            raise
+        except BleakError as exc:
+            try:
+                iface._handle_transient_read_error(exc)
+                return False, False
```

Ruff (0.15.5)

[warning] Consider moving this statement to an `else` block

(TRY300)